PIL/Image.py

```diff
@@ -1250,6 +1322,7 @@ def getpixel(self, xy):
         return self.im.getpixel(xy)
 
     def getprojection(self):
+        # type: () -> Tuple[List[Any], List[Any]]
```

Comment:
I think we can constrain this return type to `Tuple[List[int], List[int]]` ~~, but I'm confused a little by what `_binary.i8` actually does.~~ I wasn't aware that in Python 2, `bytes is str`.